hangups/default.conf

```diff
@@ -0,0 +1,11 @@
+[General]
+col-scheme: default
+
+[Keybindings]
+key-prev-tab: ctrl u
+key-next-tab: ctrl d
+key-close-tab: ctrl w
+key-quit: ctrl e
+key-menu: ctrl n
+key-up: k
+key-down: j
```

Comment:
Should this file be in the root of the repo instead? Usually that's going to be the working directory, which is where hangups will look for it.